include/open62541/server.h

```diff
@@ -1738,6 +1738,31 @@ UA_Server_deleteCondition(UA_Server *server,
                           const UA_NodeId condition,
                           const UA_NodeId conditionSource);
 
+/*
+* Set the LimitState of the LimitAlarmType
+*
+* @param server The server object
+* @param conditionId The NodeId of the node representation of the Condition Instance
+* @param limitValue The value from the trigger node
+*/
+UA_StatusCode UA_EXPORT
+UA_Server_setLimitState(UA_Server *server, const UA_NodeId conditionId,
+                        UA_Double limitValue);
+
+#if defined(UA_ENABLE_ENCRYPTION) && \
```

Comment:
That's too much ifedeffery.

UA_Server_setExpirationDate can be always defined.
If encryption is disabled (or the pointer inside certificateVerification is NULL), log and return an error code.